launch/Jenkinsfile

```diff
@@ -174,7 +173,6 @@ def releaseOpenHabComponent(componentName, branch, releaseVersion, nextVersion,
                             sh "mvn versions:update-parent -DallowSnapshots=true -DparentVersion=[" + env.OH2_NEXT_VERSION +"-SNAPSHOT] " + mvnOptions
                         }
                         if(updateProperties) {
-                            sh "mvn versions:set-property -Dproperty=esh.version -DnewVersion="+ env.ESH_NEXT_VERSION +"-SNAPSHOT " + mvnOptions
```

Comment:
@wborn, would it be related if I now get this error:

```
openhab> feature:install esh.tp-commons-net
Error executing command: No matching features for esh.tp-commons-net/0
```

You suggested installing that https://github.com/openhab/openhab2-addons/issues/4709#issuecomment-461855772

Reply:
It's unrelated because this line was just for updating the (no longer existing) `esh.version` property in case of a release.